glimr/search.py

```diff
@@ -106,6 +108,7 @@ def __init__(
         stopper=None,
         metric=None,
         mode="max",
+        cv=False,
```

Comment:
Another option would be to have the user specify the number of folds here (default None), and do the work of altering the config for the user here in search. For example, performing the modifications of `data_space` above within the search function below. Then we just document what arguments the data loader needs to accept if using CV.